sklearn/model_selection/_split.py

```diff
@@ -1894,6 +1898,9 @@ def check_cv(cv=3, y=None, classifier=False):
         splits via the ``split`` method.
     """
     if cv is None:
+        warnings.warn("The default value of n_splits=3 is deprecated"
```

Comment:
The default value of cv on line 1866 should be cv=None, otherwise we will never enter this block.

Reply:
Done. Thanks!

Reply:
maybe pass an explicit value for cv?